src/components/Profile/ProfileContainer.js

```diff
@@ -10,7 +11,12 @@ import Profile from './Profile';
 
 class ProfileContainer extends Component {
   componentDidMount() {
-    axios.get(PROFILE_URL)
+    let { userId } = this.props.match.params;
+    if (!userId) {
+      userId = 2;
```

Comment:
Consider storjng the default user ID in the core/constants of your project.

Reply:
/fxd